Source/WebCore/rendering/RenderElement.h

```diff
@@ -409,6 +409,8 @@ class RenderElement : public RenderObject {
     void updateReferencedSVGResources();
     void clearReferencedSVGResources();
 
+    bool isTopLayerAncestor();
```

Comment:
should be `const`. The name makes me pause a bit; perhaps clearer as `isAncestorOfTopLayerElement`

Reply:
Right I missed that, new patch removes this code now.